src/ui/CategoryFacet/CategoryFacetTemplates.ts

```diff
@@ -0,0 +1,42 @@
+import { $$, Dom } from '../../utils/Dom';
+import { SVGIcons } from '../../utils/SVGIcons';
+import { SVGDom } from '../../utils/SVGDom';
+import _ = require('underscore');
+
+export interface CategoryFacetData {
+  value: string;
+  count: number;
+}
+
+export class CategoryFacetTemplates {
+  private listRoot: Dom;
+  private collapseArrow: Dom;
+
+  constructor() {
+    this.listRoot = $$('ol', { className: 'coveo-facet-values' });
```

Comment:
Normal facets are `ul / li`. 

Would it be better to use the same element for simplicity reason, or if people want to customize it, they will need less css ?